### PR TITLE
New: Advanced settings toggle in indexer edit/add modal

### PR DESCRIPTION
**Pull request by mynameisbogdan**
(cherry picked from commit a570fd2a8f138fb89c5b0ae43ddb7919f6bf26e3)

#### Database Migration
NO

### CODE REVIEW

File: frontend/src/Settings/AdvancedSettingsButton.js [hunk 1]
```diff
@@ -4,18 +4,20 @@ import React from 'react';
...
-    onAdvancedSettingsPress
+    onAdvancedSettingsPress,
+    showLabel
   } = props;
 
   return (
     <Link
       className={styles.button}
-      title={advancedSettings ? 'Shown, click to hide' : 'Hidden, click to show'}
+      title={advancedSettings ? translate('ShownClickToHide') : translate('HiddenClickToShow')}
```
Comment (markus101) [inline frontend/src/Settings/AdvancedSettingsButton.js:20]: Strings in translate should be the English string, not a token.

File: frontend/src/Settings/AdvancedSettingsButton.js [hunk 2]
```diff
@@ -42,18 +44,27 @@ function AdvancedSettingsButton(props) {
...
-      <div className={styles.labelContainer}>
-        <div className={styles.label}>
-          {advancedSettings ? 'Hide Advanced' : 'Show Advanced'}
-        </div>
-      </div>
+      {
+        showLabel ?
+          <div className={styles.labelContainer}>
+            <div className={styles.label}>
+              {advancedSettings ? translate('HideAdvanced') : translate('ShowAdvanced')}
```
Comment (markus101) [inline frontend/src/Settings/AdvancedSettingsButton.js:51]: Same thing here

**Review by markus101 — CHANGES_REQUESTED** [review]

File: frontend/src/Settings/AdvancedSettingsButton.js [hunk 3]
```diff
@@ -42,18 +43,27 @@ function AdvancedSettingsButton(props) {
...
-      <div className={styles.labelContainer}>
-        <div className={styles.label}>
-          {advancedSettings ? 'Hide Advanced' : 'Show Advanced'}
-        </div>
-      </div>
+      {
+        showLabel ?
+          <div className={styles.labelContainer}>
+            <div className={styles.label}>
+              {advancedSettings ? 'Hide Advanced' : 'Show Advanced'}
```
Comment (Qstick) [inline frontend/src/Settings/AdvancedSettingsButton.js:50]:
```suggestion
              {advancedSettings ? translate('Hide Advanced') : translate('Show Advanced')}
```

Comment (markus101) [reply frontend/src/Settings/AdvancedSettingsButton.js:50]: I think this was completely removed based on this comment: https://github.com/Sonarr/Sonarr/pull/5646#pullrequestreview-1425724887

We had some further discussion in https://github.com/Sonarr/Sonarr/pull/5640 which makes a lot of sense for consistency across the different *arr apps.

**Pull request merged**